index.js

```diff
@@ -28,6 +28,17 @@ function TestExclude (opts) {
     this.include = false
   }
 
+  // unless someone explicitly provides !node_modules
+  // always ignore node_modules.
+  if (
+    !this.exclude.filter(function (e) {
+      return /!node_modules/.test(e)
```

Comment:
This looks very fragile and `micromatch` already handles negation globs: https://github.com/jonschlinkert/micromatch#usage-examples